extensions/tekton/examples/src/main/java/io/fabric8/tekton/api/examples/CancelTask.java

```diff
@@ -0,0 +1,31 @@
+package io.fabric8.tekton.api.examples;
```

Comment:
License header is missing.

Reply:
Thanks @oscerd . Does the license have to be anything in particular? If I look at another example it mentions RedHat in the CopyRight? https://github.com/fabric8io/kubernetes-client/blob/master/extensions/tekton/examples/src/main/java/io/fabric8/tekton/api/examples/TaskCreate.java